pytorch_lightning/trainer/connectors/accelerator_connector.py

```diff
@@ -550,6 +557,11 @@ def _is_slurm_managing_tasks(self) -> bool:
     def _choose_strategy(self) -> Union[Strategy, str]:
         if self._accelerator_flag == "ipu":
             return IPUStrategy.strategy_name
+        if self._accelerator_flag == "hpu":
+            if self._parallel_devices and len(self._parallel_devices) > 1:
+                return HPUParallelStrategy(parallel_devices=self.parallel_devices)  # type: ignore
```

Comment:
```suggestion
                return HPUParallelStrategy.strategy_name
```

The single_device is returning a class here because the `device` is not lazy initialized, we will fix it. 
`parallel_devices` is lazy initialized, we set in the `_lazy_init_strategy`. Let's keep this way to be consistent with rest of strategies?